enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+      effectuate this change, you must delete the service: `oc -n openshift-ingress
+      delete svc/router-test`; the service load-balancer will then be deprovisioned
+      and a new one created.  This will most likely cause the new load-balancer to
+      have a different host name and IP address from the old one''s.  Alternatively,
+      you can revert the change to the IngressController:  `oc -n openshift-ingress-operator
+      patch ingresscontrollers/test --type=merge --patch=''{"spec":{"endpointPublishingStrategy":{"loadBalancer":{"providerParameters":{"type":"AWS","aws":{"type":"NLB","eipAllocations":["eipalloc-0956fea34de4cb7ab","eipalloc-0e9a3077a70de050a","eipalloc-0b69fc4691f54cdd0","eipalloc-01e6ba6cbba1a391b","eipalloc-0e242df173f906112"]}}}}}}'''
+       reason: IngressControllerProgressing
+       status: "True"
+       type: Progressing
+     - lastTransitionTime: "2024-04-10T00:20:50Z"
+       status: "False"
+       type: Degraded
+     - lastTransitionTime: "2024-04-10T00:20:17Z"
+       message: IngressController is upgradeable.
+       reason: Upgradeable
+       status: "True"
+       type: Upgradeable
+     - lastTransitionTime: "2024-04-10T00:20:17Z"
+       message: No evaluation condition is detected.
+       reason: NoEvaluationCondition
+       status: "False"
+       type: EvaluationConditionsDetected
+   domain: apps1.apps.misalunkeip16.devcluster.openshift.com
+   endpointPublishingStrategy:
+     loadBalancer:
+       dnsManagementPolicy: Managed
+       providerParameters:
+         aws:
+           networkLoadBalancer:
+             eipAllocations:
+               - eipalloc-0956fea34de4cb7ab
+               - eipalloc-0e9a3077a70de050a
+               - eipalloc-0b69fc4691f54cdd0
+               - eipalloc-01e6ba6cbba1a391b
+               - eipalloc-0e242df173f906112
+           type: NLB
+         type: AWS
+       scope: External
+     type: LoadBalancerService
+   observedGeneration: 3
+   selector: ingresscontroller.operator.openshift.io/deployment-ingresscontroller=test
+   tlsProfile:
+     ciphers:
+       - ECDHE-ECDSA-AES128-GCM-SHA256
+       - ECDHE-RSA-AES128-GCM-SHA256
+       - ECDHE-ECDSA-AES256-GCM-SHA384
+       - ECDHE-RSA-AES256-GCM-SHA384
+       - ECDHE-ECDSA-CHACHA20-POLY1305
+       - ECDHE-RSA-CHACHA20-POLY1305
+       - DHE-RSA-AES128-GCM-SHA256
+       - DHE-RSA-AES256-GCM-SHA384
+       - TLS_AES_128_GCM_SHA256
+       - TLS_AES_256_GCM_SHA384
+       - TLS_CHACHA20_POLY1305_SHA256
+     minTLSVersion: VersionTLS12
+
+```
+
+#### Cluster admin wants operator to update the Load Balancer service for updated Ingress Controller eipAllocations 
+
+Administrator can tell the Ingress Operator to complete the updating of Load Balancer service automatically
+by annotating the IngressController with the annotation called `ingress.operator.openshift.io/auto-delete-load-balancer=` 
```

Comment:
Does this annotation really have `=` at the end?

Reply:
yes

Reply:
but to provide a blank value we have to put = and nothing on the right side of it